Libbna/CUSTOM-CMS
# #258: Fixed card image height issue

**Testing Steps:**

- Add any article with an image.
- Go to the homepage and check if the height of the new is the same as others.
- Refer to the following screenshot.
- Also check for the hover effects on image.

![image](https://user-images.githubusercontent.com/81369580/151927299-d3c13919-6002-4b47-899e-07c0d03eb1f1.png)
